src/main/java/uk/gov/hmcts/befta/util/CucumberStepAnnotationUtils.java

```diff
@@ -0,0 +1,172 @@
+package uk.gov.hmcts.befta.util;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.Map;
+
+public final class CucumberStepAnnotationUtils {
+
+    // Hide Utility Class Constructor : Utility classes should not have a public or default constructor (squid:S1118)
+    private CucumberStepAnnotationUtils() { }
+
+    public static void adjustCucumberStepAnnotations(Class<?> testClass) {
```

Comment:
please rename this to --> injectCommonSyntaticFlexibilitiesIntoStepDefinitionsIn
This name will tell more directly what the method is doing.

Reply:
The function name has now been updated to: `injectCommonSyntacticFlexibilitiesIntoStepDefinitions`